### PR TITLE
Dev sum potential

### PR DESCRIPTION
**Pull request by stenczelt**
sum potential calculator initalisation, changed the signature of the Potetial class, a little bugfix and a test for the calculator
see #184

### CODE REVIEW

File: quippy/quippy/potential.py [hunk 1]
```diff
@@ -71,10 +73,21 @@ def __init__(self, args_str="", param_str=None, atoms=None, calculation_always_r
...
         # init the quip potential
         if param_filename is not None and type(param_filename) == str:
+            # from a param filename
             self._quip_potential = quippy.potential_module.Potential.filename_initialise(args_str=args_str,
                                                                                          param_filename=param_filename)
-        else:
+        elif param_str is not None:
+            # from a param string
             self._quip_potential = quippy.potential_module.Potential(args_str=args_str, param_str=param_str)
+        elif pot1 is not None and pot2 is not None:
```
Comment (jameskermode) [inline quippy/quippy/potential.py:82]: There are other ways to construct a Potential that accept `pot1` and `pot2`, e.g. a QM/MM-style `ForceMixingPotential`. Might be worth checking if this code works correctly for that use case too.

**Comment by jameskermode** [comment]
Looks good to me. If you have time/willingness to try a ForceMixingPotential (we could revive one of the old tests from the Py2.7 branch) that would be great, otherwise feel free to go ahead and merge as is.

File: quippy/quippy/potential.py [hunk 2]
```diff
@@ -44,24 +44,26 @@ class Potential(ase.calculators.calculator.Calculator):
     callback_map = {}
 
     implemented_properties = ['energy', 'free_energy', 'forces', 'virial', 'stress',
-                              'local_virial', 'local_energy', 'local_stress']
+                              'local_virial', 'local_energy', 'local_stress', 'stresses', 'energies']
```
Comment (jameskermode) [inline quippy/quippy/potential.py]: We could consider if we should remove `local_stress` altogether - `stresses` is the expected ASE-compatible name.

File: quippy/quippy/potential.py (hunk 1 above)
Comment (stenczelt) [reply quippy/quippy/potential.py:82]: I took a look at the py2 code and the currently wrapped one as well. we have `quippy.potential_module.Potential_FM` as well, so should probably use that directly. I think this needs more insight and time to develop a reliable wrapper for, which we could do together with the DynamicalSystems, that we have started but left unfinished at the end of my last visit

File: quippy/quippy/potential.py (hunk 2 above)
Comment (stenczelt) [reply quippy/quippy/potential.py]: ok

**Pull request merged**